applications/northstar/main.tf

```diff
@@ -35,6 +35,15 @@ variable "name" {
   description = "The application name."
 }
 
+variable "storage_name" {
+  description = "The storage bucket name."
+}
```

Comment:
This is necessary since the bucket's name (e.g. `dosomething-rogue`) will no longer match the application name (e.g. `dosomething-northstar`). We can consider cleaning this up later, but it'd be a timely migration since you can't rename existing buckets.